src/main/java/com/griddynamics/controllers/ProductController.java

```diff
@@ -39,15 +44,19 @@ public ProductController(ProductService productService) {
         ProductDTO product;
 
         try {
+            log.info("Trying get product by id " + id);
```

Comment:
dont concatenate strings.
logger provides the elegant way to do this
check some like that
 log.info("Trying get product by id {} ", id);